bindings/core/src/method/wallet.rs

```diff
@@ -194,7 +195,11 @@ pub enum WalletMethod {
     /// Prepares to transition an implicit account to an account.
     /// Expected response: [`PreparedTransaction`](crate::Response::PreparedTransaction)
     #[serde(rename_all = "camelCase")]
-    PrepareImplicitAccountTransition { output_id: OutputId },
+    PrepareImplicitAccountTransition {
+        output_id: OutputId,
+        public_key: Option<PublicKey>,
+        bip_path: Option<Bip44>,
```

Comment:
So if `public_key` is set the value in `bip_path` is ignored no matter if it's Some or None?

Could this be a merged like this for example:
```rust
key_source: Option<KeySource>
```

Reply:
okay :+1: